### DISCUSSION TITLE
Merging character LoRAs: how about the trigger words?

### DISCUSSION THREAD

**Discussion by G-Grifflu** [bmaltais/kohya_ss#1623] [Q&A] [answered]
I want to merge 2 SDXL character LoRAs to create a new hybrid character.
Each LoRA has its own trigger words, based on famous people known by the SDXL model and that look relatively close to the characters, e.g. "Tom Cruise man".
If I merge these LoRAs, what become of the trigger words, and how do I trigger the new merged LoRA character?

**Comment by TomLucidor** [comment, marked as answer]
Classic rule is to weight each as 0.5 each, assuming you want an average. They do remember each form of the trigger words, so weight them wisely
